src/sys/socket/sockopt.rs

```diff
@@ -858,14 +866,34 @@ sockopt_impl!(
 #[cfg(feature = "net")]
 sockopt_impl!(
     #[cfg_attr(docsrs, doc(cfg(feature = "net")))]
-    /// Set delivery of the `IPV6_PKTINFO` control message on incoming
+    /// Set delivery of the `IPV6_RECVPKTINFO` control message on incoming
     /// datagrams.
     Ipv6RecvPacketInfo,
     Both,
     libc::IPPROTO_IPV6,
     libc::IPV6_RECVPKTINFO,
     bool
 );
+#[cfg(any(
+    target_os = "android",
+    target_os = "freebsd",
+    target_os = "ios",
+    target_os = "linux",
+    target_os = "macos",
+    target_os = "netbsd",
+    target_os = "openbsd",
+))]
```

Comment:
This symbol seems to be available on DragonFlyBSD and other Apple targets, if we add them, then this can be changed to:

```suggestion
#[cfg(any(linux_android, bsd))]
```

Reply:
Updated!